Graphs/Table/index.js

```diff
@@ -306,6 +306,8 @@ const TableGraph = (props) => {
                 const columnRow = columns[index];
                 const displayColumn = stateColumn.includes(columnRow.label || columnRow.column) ? 'false' : 'true';
                 const sort = !isEmpty(columnKeys) ? columnKeys.get(columnRow.column) : true;
+                // save typeof column value if additionalSearchOperator is true
+                const columnDataType = !!properties.additionalSearchOperator && !!data.length ? typeof objectPath.get(data[0], columnRow.column) : undefined;
```

Comment:
I see a basic flaw in the present design. The additional numerical operators will work only if current table data is non-empty.
In the below example packet count in all rows is 10. I use a filter > 33, which results in empty table. After this, the numeric operators will no more be listed as data is empty array now. This is very flaky logic.

![tablesearch](https://user-images.githubusercontent.com/24920919/108439480-31742e80-7206-11eb-8928-0b0f63021c86.gif)

Instead of using data[0], in case `enableNumericSearch` is `true`, we must issue a `_mapping` request and extract the type from response,

<img width="586" alt="Screen Shot 2021-02-18 at 4 32 01 PM" src="https://user-images.githubusercontent.com/24920919/108439876-f7575c80-7206-11eb-9915-2c428900ad4e.png">

There should be only one `_mapping` request issued per ES index.